compiler/rustc_middle/src/traits/solve.rs

```diff
@@ -1,97 +1,24 @@
 use rustc_ast_ir::try_visit;
 use rustc_data_structures::intern::Interned;
 use rustc_macros::{HashStable, TypeFoldable, TypeVisitable};
-use rustc_span::def_id::DefId;
+use rustc_next_trait_solver::solve as ir_solve;
```

Comment:
```suggestion
use rustc_next_trait_solver as ir;
```
and use `ir::solve::Whatever`, feels a bit cleaner to me